src/query/time_to_live/time_to_live.cpp

```diff
@@ -0,0 +1,291 @@
+    }
+    {
+      const auto per = storage_.Get("period");
+      if (!per) {
+        return fail("period");
+      }
+      if (per->empty())
+        info_.period = std::nullopt;
+      else
+        info_.period = TtlInfo::ParsePeriod(*per);
+    }
+    {
+      const auto st = storage_.Get("start_time");
+      if (!st) {
+        return fail("start_time");
+      }
+      if (st->empty())
+        info_.start_time = std::nullopt;
+      else
+        info_.start_time = TtlInfo::ParseStartTime(*st);
+    }
+    {
+      const auto run = storage_.Get("running");
+      if (!run || (*run != "false" && *run != "true")) {
+        return fail("running");
+      }
+      if (*run == "true") {
+        Setup_(db, interpreter_context);
+      }
+    }
+  } catch (TtlException &e) {
+    return fail(e.what());
+  }
+  return true;
+}
+
+std::chrono::microseconds TtlInfo::ParsePeriod(std::string_view sv) {
+  if (sv.empty()) return {};
+  utils::DurationParameters param;
+  int val = 0;
+  for (const auto c : sv) {
+    if (isdigit(c)) {
+      val = val * 10 + (int)(c - '0');
+    } else {
+      switch (tolower(c)) {
+        case 'd':
+          param.day = val;
+          break;
+        case 'h':
+          param.hour = val;
+          break;
+        case 'm':
+          param.minute = val;
+          break;
+        case 's':
+          param.second = val;
+          break;
+        default:
+          throw TtlException("Badly defined period. Use integers and 'd', 'h', 'm' and 's' to define it.");
+      }
+      val = 0;
+    }
+  }
+  return std::chrono::microseconds{utils::Duration(param).microseconds};
+}
+
+// We do not support microseconds, but are aligning to the timestamp() values
+std::string TtlInfo::StringifyPeriod(std::chrono::microseconds us) {
+  std::string res;
+  if (const auto di = GetPart<std::chrono::days>(us)) {
+    res += fmt::format("{}d", di);
+  }
+  if (const auto hi = GetPart<std::chrono::hours>(us)) {
+    res += fmt::format("{}h", hi);
+  }
+  if (const auto mi = GetPart<std::chrono::minutes>(us)) {
+    res += fmt::format("{}m", mi);
+  }
+  if (const auto si = GetPart<std::chrono::seconds>(us)) {
+    res += fmt::format("{}s", si);
+  }
+  return res;
+}
+
+/**
+ * @brief From user's local time to system time. Uses timezone
+ *
+ * @param sv
+ * @return std::chrono::system_clock::time_point
+ */
+std::chrono::system_clock::time_point TtlInfo::ParseStartTime(std::string_view sv) {
+  try {
+    // Midnight might be a problem...
+    const auto now =
+        std::chrono::year_month_day{std::chrono::floor<std::chrono::days>(std::chrono::system_clock::now())};
+    const utils::DateParameters date{static_cast<int>(now.year()), static_cast<unsigned>(now.month()),
+                                     static_cast<unsigned>(now.day())};
+    auto [time, _] = utils::ParseLocalTimeParameters(sv);
+    const utils::ZonedDateTimeParameters zdt{date, time, utils::Timezone(std::chrono::current_zone()->name())};
```

Comment:
```suggestion
    const utils::ZonedDateTimeParameters zdt{date, time, utils::Timezone(std::chrono::current_zone()->name())};
```
Use fixed LocalDateTime

Reply:
+ update query/frontend/stripped_lexer_constants.hpp